src/narjure/memory_management/persistence_manager.clj

```diff
@@ -22,18 +51,19 @@
   (shutdown!))
 
 (defn initialise
-  "Initialises actor:
-      registers actor and sets actor state"
+  "Initialises actor: registers actor and sets actor state"
   [aname actor-ref]
   (register! aname actor-ref)
-  (set-state! {:state 0}))
+  (set-state! {:path "d:/persistence/snapshot.nar" :concept-count 0 :received-states 0}))
```

Comment:
`d:/persistence/snapshot.nar` it should be moved somewhere, we can use something like https://github.com/weavejester/environ for such things